src/__tests__/streams.ts

```diff
@@ -0,0 +1,9 @@
+import { Readable } from 'stream';
+import { sdkStreamMixin } from '@smithy/util-stream';
+
+export function stringToStream(s: string): any {
```

Comment:
Can we type this correctly?
```suggestion
export function stringToStream(s: string): any {
```